File: apps/hotel-booking-service-admin/src/api/customer/CustomerUpdateInput.ts
import { BookingUpdateManyWithoutCustomersInput } from "./BookingUpdateManyWithoutCustomersInput";

export type CustomerUpdateInput = {
  bookings?: BookingUpdateManyWithoutCustomersInput;
  description?: string | null;
  email?: string | null;
  name?: string | null;
  phoneNumber?: string | null;
};

File: apps/dotnet-service/src/APIs/Customer/Dtos/CustomerUpdateInput.cs
namespace DotnetService.APIs.Dtos;

public class CustomerUpdateInput
{
    public List<string>? Bookings { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Description { get; set; }

    public string? Email { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public string? PhoneNumber { get; set; }

    public DateTime? UpdatedAt { get; set; }
}
